src/joint.dia.element.js

```diff
@@ -775,7 +776,6 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
             // let the linkview deal with this event
             this._linkView.pointerup(evt, x, y);
-
             delete this._linkView;
 
```

Comment:
and `batch:stop` here. Once I merge this PR I'm going to add names for the batches. e.g.

    this.model.trigger('batch:stop', { batchName: 'add-link' });

which might be useful not only for debugging purposes.